README.md

```diff
@@ -1,25 +1,30 @@
-<img src="http://bitcore.io/css/images/module-mnemonic.png" alt="bitcore mnemonics" height="35">
-BIP39 Mnemonics for bitcore
+Bitcore Mnemonics
 =======
 
 [![NPM Package](https://img.shields.io/npm/v/bitcore-mnemonic.svg?style=flat-square)](https://www.npmjs.org/package/bitcore-mnemonic)
 [![Build Status](https://img.shields.io/travis/bitpay/bitcore-mnemonic.svg?branch=master&style=flat-square)](https://travis-ci.org/bitpay/bitcore-mnemonic)
 [![Coverage Status](https://img.shields.io/coveralls/bitpay/bitcore-mnemonic.svg?style=flat-square)](https://coveralls.io/r/bitpay/bitcore-mnemonic)
 
-A module for [bitcore](https://github.com/bitpay/bitcore) that implements [Mnemonic code for generating deterministic keys](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki).
+A module for [bitcore](https://github.com/bitpay/bitcore) that implements [Mnemonic code for generating deterministic keys](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) (BIP39).
 
 ## Getting Started
 
 This library is distributed in both the npm and bower packaging systems.
 
 ```sh
+npm install bitcore-lib
```

Comment:
Likely not directly, however I think it's good to do so that dependencies are setup already.